src/main/java/Dan/Dan.java

```diff
@@ -0,0 +1,53 @@
+package Dan;
+
+import java.io.IOException;
+import java.nio.file.NoSuchFileException;
+
+import java.util.Scanner;
+
+
+public class Dan {
```

Comment:
Perhaps you forgot java docs for this class and its methods?

Reply:
Thanks for spotting this!